src/fittable_laws.jl

```diff
@@ -77,3 +77,172 @@ function (law::FM90)(wave::T) where T
     return exvebv
 end
 
+
+"""
+    P92(BKG_amp=218.57,
```

Comment:
You'll want to show that these are keyword arguments with the semicolon
```julia
    P92(; BKG_amp=218.57,
```
(and propagate the spacing).

Reply:
Sorry, I clicked on resolved by mistake. 😅